apps/foosball-api/project.json

```
@@ -48,6 +48,23 @@
         "jestConfig": "apps/foosball-api/jest.config.ts",
         "passWithNoTests": true
       }
+    },
+    "dockerize": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "docker build -f ./apps/foosball-api/server/Dockerfile . -t foosball-api"
+      },
```

Would've expected the working _dir_ to be this app folder already. But probably not the case looking at this code.
Would ... be an improvement to set the [cwd](https://nx.dev/packages/workspace/executors/run-commands#cwd)?
(Haven't tested it locally)
```suggestion
      "options": {
        "cwd": "apps/foosball-api",
        "command": "docker build -f ./server/Dockerfile . -t foosball-api"
      },
```

---

Oops, noticed that this change will break the script. When changing the 'cwd', the path references in the Dockerfile also break. 

So will consider this to be an improvement for 'the future'.